inst/reports/wgts-qc/render.sh

```
@@ -0,0 +1,8 @@
+date1="2024-12-03"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)